gatsby-node.ts

```diff
@@ -38,7 +38,8 @@ const commonRedirectProps = {
  * @returns boolean for if file is outdated or not
  */
 const checkIsMdxOutdated = (filePath: string): boolean => {
-  const dirname = path.resolve("./")
+  // .replace(/\\/g, "/") to replace \ in windows paths ex: C:\\folder\\myfile.txt becomes C:/folder/myfile.txt
+  const dirname = path.resolve("./").replace(/\\/g, "/")
```

Comment:
There are a few other places were we use `path.resolve`. Wondering why we wouldn't have this problem in those cases.

Also, `path.resolve` takes care of this problem https://nodejs.org/api/path.html#windows-vs-posix

I'm trying to understand why we need to do this and why this is not breaking in other places.